packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -69,6 +43,28 @@ function APIProvider({ children }) {
       storyLocking,
       pageTemplates: customPageTemplates,
     },
+    config: {
+      getStoryById,
+      getStoryLockById,
+      setStoryLockById,
+      deleteStoryLockById,
+      getDemoStoryById,
+      saveStoryById,
+      autoSaveById,
+      getMedia,
+      uploadMedia,
+      updateMedia,
+      deleteMedia,
+      getLinkMetadata,
+      getAuthors,
+      getCurrentUser,
+      updateCurrentUser,
+      saveMetaBoxes,
```

Comment:
General note / reminder that we should move all meta box related functionality to `wp-story-editor`, including `useSaveMetaBoxes`. This API provider should not know about meta boxes.